### ISSUE TITLE
@auth works?

### ISSUE THREAD

**Issue by conanchen** [dgraph-io/dgraph#5610]
### What version of Dgraph are you using?
20.03.3
### Have you tried reproducing the issue with the latest release?
the latest

### What is the hardware spec (RAM, OS)?
macos

### Steps to reproduce the issue (command/config used to run Dgraph).
1. just try to update schema with https://raw.githubusercontent.com/dgraph-io/dgraph/master/graphql/schema/testdata/schemagen/input/authorization.graphql
2. got schema update error
```
$ curl -X POST localhost:8080/admin/schema -d '@build/authorization.graphql'

{"errors":[{"message":"couldn't rewrite mutation updateGQLSchema because input:1: Undefined directive auth.\n","extensions":{"code":"Error"}}]}
```

### Expected behaviour and actual result.
update schema sucessfully, actual got error.

**Comment by conanchen** [comment]
@arijitAD thanks, 

Is @auth feature part of Enterprise version?

**Comment by arijitAD** [comment]
AFAIK it will be available in the community version in v20.07. It's also available in Slash GraphQL.
https://discuss.dgraph.io/t/public-a-memo-about-slash-graphql/7100
